meson.build

```diff
@@ -722,6 +722,8 @@ if time_epoch == -1
 endif
 conf.set('TIME_EPOCH', time_epoch)
 
+conf.set('CLOCK_DIFF_THRESHOLD', get_option('clock-diff-threshold'))
```

Comment:
not a fan of needless abbreviations, i.e. "diff" is not pretty.

also, this should make clear in its naming that this is a *maximum* threshold that is applied.

i.e. CLOCK_VALID_RANGE_MAX or so.